ui/src/app/applications/components/application-resources-diff/application-resources-diff.tsx

```diff
@@ -33,10 +33,13 @@ export const ApplicationResourcesDiff = (props: ApplicationResourcesDiffProps) =
                 return {
                     a: live ? jsYaml.safeDump(live, {indent: 2}) : '',
                     b: target ? jsYaml.safeDump(target, {indent: 2}) : '',
+                    hook: state.liveState.metadata.annotations['argocd.argoproj.io/hook'],
```

Comment:
I'm aware that this will not deal with Helm, but it'll do for now, and Helm can be fixed in #1865.